projects/components/src/datagrid/datagrid.component.ts

```diff
@@ -218,6 +242,8 @@ export class DatagridComponent<R> implements OnInit {
         return this._buttonConfig;
     }
 
+    constructor(private component: ElementRef) {}
```

Comment:
component is misleading, it's an html element, a node. A component is an abstraction over a node